src/utils/processArguments.js

```diff
@@ -0,0 +1,27 @@
+const argv = require('yargs')
+    .option('o', {
+        alias: 'output',
+        describe: 'Output file path with miragejs generated code.',
+        type: 'string',
+        nargs: 1,
+        demand: true,
+        demand: 'An output path is required',
+        default: './miragejs.js'
+    })
+    .option('i', {
+        alias: 'input',
+        describe: 'Input file path/url with open api specification.',
+        type: 'string',
+        nargs: 1,
+        demand: true,
+        demand: 'An input path/url is required',
+    }).argv
+
+function getProcessArguments() {
+    return argv;
```

Comment:
I'd love the yargs code here, for one reason:

Imagine you require this file somewhere else, even if you then don't run this function, the yargs code is still executed, even if nothing is done. 

If you move it here, it only runs when this function is called, what do you think?

Reply:
Agree. Moved it inside.